Strategy11/formidable-forms
# Include term_parent in exported xml

Fix https://github.com/Strategy11/formidable-pro/issues/799

### Test steps
1. Create a form with and configure a 'Create Post' action in such a way that Category taxonomy is mapped to a field in the form.
2. Preview the form and select Category that has a Parent category.
3. Export the form xml along with entries.
4. Open the export xml file and check that the parent category is found like `<term_parent>parent_id</term_parent>`
